exporter/awsemfexporter/datapoint.go

```diff
@@ -61,10 +61,10 @@ type dataPoints interface {
 	// retained: indicates whether the data point is valid for further process
 	// NOTE: It is an expensive call as it calculates the metric value.
 	CalculateDeltaDatapoints(i int, instrumentationScopeName string, detailedMetrics bool, calculators *emfCalculators) (dataPoint []dataPoint, retained bool)
-	// IsStaleOrNaN returns true if metric value has NoRecordedValue flag set or if any metric value contains a NaN.
-	// When return value is true, IsStaleOrNaN also returns the attributes attached to the metric which can be used for
+	// IsStaleOrNaNOrInf returns true if metric value has NoRecordedValue flag set or if any metric value contains a NaN or Inf.
+	// When return value is true, IsStaleOrNaNOrInf also returns the attributes attached to the metric which can be used for
 	// logging purposes.
-	IsStaleOrNaN(i int) (bool, pcommon.Map)
+	IsStaleOrNaNOrInf(i int) (bool, pcommon.Map)
```

Comment:
```suggestion
	IsStaleNaNInf(i int) (bool, pcommon.Map)
```

nit: naming/readability preference.